### ISSUE TITLE
create a target test gem

### ISSUE THREAD

**Issue by emig** [CodePadawans/ataru#45]
In order to test the setup of ataru + target project via Bundler, (#43, #37)
a target consisiting in a test gem would be a first initial step. The gem called `padawan_moon` will declare a
module `Galaxy` with a method `answer_to_the_ultimate_question_of_life_the_universe_and_everything`
that will return the number **42**

This gem will be used to explore our assumptions on how to use Bundler to load
in ataru the dependencies of the tarjet project

**Comment by emig** [comment]
Official guide on gems http://guides.rubygems.org/make-your-own-gem/

**Comment by emig** [comment]
We have a candidate https://github.com/CodePadawans/padawans_moon, the TODO
section on the gemspec needs to be written, othwerise I get this:

```
WARNING:  See http://guides.rubygems.org/specification-reference/ for help
ERROR:  While executing gem ... (Gem::InvalidSpecificationException)
"FIXME" or "TODO" is not a description
```

**Comment by emig** [comment]
Update Readme as well, this gem won't be distributed by 
a gem server, but required with a specific git path:

```
gem 'padawans_moon', :git => 'https://github.com/CodePadawans/padawans_moon.git'
```

Bundler will take care of building and [loading it](http://bundler.io/v1.7/gemfile.html)

**Comment by moonglum** [comment]
Yep, sounds really good!

**Comment by emig** [comment]
Gut, 

I build the gem!: 

```
$ gem build padawans_moon.gemspec            
WARNING:  no homepage specified                                                                                        
WARNING:  description and summary are identical                                                                      
WARNING:  See http://guides.rubygems.org/specification-reference/ for help                                           
  Successfully built RubyGem                                                                                           
  Name: padawans_moon                                                                                                  
  Version: 0.0.1                                                                                                       
  File: padawans_moon-0.0.1.gem
```

And installed it!: 

```
$ gem install padawans_moon-0.0.1.gem        
Successfully installed padawans_moon-0.0.1                                    
Parsing documentation for padawans_moon-0.0.1                                 
Installing ri documentation for padawans_moon-0.0.1                           
Done installing documentation for padawans_moon after 0 seconds               
1 gem installed
```

But the method is not returning 42 :( 

```
$ irb
2.1.0 :001 > require 'padawans-moon'                                                                                  
2.1.0 :006 >   class A; include Galaxy; end                                                                            
 => A                                                                                                                  
2.1.0 :007 > a = A.new                                                                                                 
 => #<A:0x000001013062c0>                                                                                              
2.1.0 :008 > a.answer_to_the_ultimate_question_of_life_the_universe_and_everything                                     
42                                                                                                                     
 => nil                                                                                                                
```

We need it to return a value in order to make a method easy to test. When we
print a value, we are actually returning nil.  https://github.com/CodePadawans/padawans_moon/blob/master/lib/padawans_moon/galaxy.rb#L3

you can try on irb to see what is the return value

```
$ irb
2.1.0 :001 > puts 'lala'
lala                   
=> nil
```

Since you are going to update the code, you can also get rid of the warnings 
(put the github page of the gem as the hompage, and change a little bit the
description :))

**Comment by emig** [comment]
When closed #37 and merged, we can update the README.md and 
explain how to use the use the test gem.
Then this issue can be closed.

**Comment by madziaf** [comment]
updated README for PadawansMoon, you can find the gem here: https://github.com/CodePadawans/padawans_moon
